force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -18,26 +18,36 @@ For this lesson, students have two options:
 Let's dive into the specifics of each operation:
 */
 trigger AccountTrigger on Account (before insert, after insert) {
-
+    TriggerHandler.run(Trigger.new);
```

Comment:
I know you said you struggled implementing the TriggerHandler framework. Try referencing the [repository](https://github.com/kevinohara80/sfdc-trigger-framework) where the TriggerHandler.cls is maintained. I think the README is helpful and explains the implementation well. 

In your trigger, you just need to define the contexts that are needed at the top and then call the run method of the handler class. 
```
trigger AccountTrigger on Account (before insert, after insert) {
    new AccountTriggerHandler().run();
```

You did a great job creating Helper classes to contain the logic. Now you just need to add a Handler class for each object that extends TriggerHandler